apps/os/app/routes.ts

```diff
@@ -18,16 +21,22 @@ export default [
     route("estates", "./routes/admin-estates.tsx"),
   ]),
 
-  // Protected routes with org/estate prefix
-  route(":organizationId/:estateId", "./routes/estate-layout.tsx", [
-    index("./routes/home.tsx"),
-    route("integrations", "./routes/integrations.tsx"),
-    route("integrations/mcp-params", "./routes/integrations.mcp-params.tsx"),
-    route("integrations/redirect", "./routes/integrations.redirect.tsx"),
-    route("estate", "./routes/estate.tsx"),
-    route("agents", "./routes/agents-index.tsx"),
-    route("agents/start-slack", "./routes/agents.start-slack.tsx"),
-    route("agents/:agentClassName/:durableObjectName", "./routes/agents.tsx"),
+  // All organization routes (with or without estate context)
+  route(":organizationId", "./routes/org-layout.tsx", [
+    // Organization-level routes
+    index("./routes/org-redirect.tsx"),
+    route("settings", "./routes/org-settings.tsx"),
+    route("team", "./routes/org-team.tsx"),
+
+    // Estate-specific routes
+    route(":estateId", "./routes/home.tsx"),
+    route(":estateId/integrations", "./routes/integrations.tsx"),
+    route(":estateId/integrations/mcp-params", "./routes/integrations.mcp-params.tsx"),
+    route(":estateId/integrations/redirect", "./routes/integrations.redirect.tsx"),
+    route(":estateId/estate", "./routes/estate.tsx"),
+    route(":estateId/agents", "./routes/agents-index.tsx"),
+    route(":estateId/agents/start-slack", "./routes/agents.start-slack.tsx"),
+    route(":estateId/agents/:agentClassName/:durableObjectName", "./routes/agents.tsx"),
```

Comment:
```suggestion
    // Estate-specific routes
    ...prefix(":estateId", [
      // Estate-specific routes
      index("./routes/home.tsx"),
      route("integrations", "./routes/integrations.tsx"),
      route("integrations/mcp-params", "./routes/integrations.mcp-params.tsx"),
      route("integrations/redirect", "./routes/integrations.redirect.tsx"),
      route("estate", "./routes/estate.tsx"),
      route("agents", "./routes/agents-index.tsx"),
      route("agents/start-slack", "./routes/agents.start-slack.tsx"),
      route("agents/:agentClassName/:durableObjectName", "./routes/agents.tsx"),
    ]),
```
dont forget import the `prefix` function